prometheus_client/values.py

```diff
@@ -51,6 +52,12 @@ class MmapedValue(object):
 
         def __init__(self, typ, metric_name, name, labelnames, labelvalues, multiprocess_mode='', **kwargs):
             self._params = typ, metric_name, name, labelnames, labelvalues, multiprocess_mode
+            # This deprecation warning can go away in a few releases when removing the compatibility
+            if 'prometheus_multiproc_dir' in os.environ and 'PROMETHEUS_MULTIPROC_DIR' in os.environ:
+                warnings.warn("You have declared both env variable in upper case and lower case, keep the upper case only PROMETHEUS_MULTIPROC_DIR'", DeprecationWarning)
```

Comment:
You have an extra single tick after PROMETHEUS_MULTIPROC_DIR here.

Reply:
Good catch!